ganttproject/src/main/resources/resources/sql/init-project-database.sql

```diff
@@ -0,0 +1,31 @@
+create table if not exists Task (
+    id                      integer         not null    primary key,
+    name                    varchar         not null,
+    color                   varchar         null,
+    shape                   varchar         null,
+    is_milestone            boolean         not null,
+    is_project_task         boolean         not null,
+    start_date              timestamp(0)    not null,
+    duration                integer         not null,
+    completion              integer         null,
+    earliest_start_date     timestamp(0)    null,
+    third_date_constraint   integer         null,
```

Comment:
this field is useless (it is always "1" when the earliest start date is set, and takes no other values).